File: core/src/main/java/com/ustadmobile/core/view/ReportBarChartComponentView.java
package com.ustadmobile.core.view;


import java.util.List;

/**
 * Component for presenter access
 */
public interface ReportBarChartComponentView extends UstadView{

    // This defines the view name that is an argument value in the go() in impl.
    String VIEW_NAME = "ReportBarChartComponentView";

    //Any argument keys:

    void setChartData(List<Object> dataSet);

}

File: core/src/commonMain/kotlin/com/ustadmobile/core/view/ReportBarChartComponentView.kt
package com.ustadmobile.core.view

/**
 * Component for presenter access
 */
interface ReportBarChartComponentView : UstadView {

    //Any argument keys:

    fun setChartData(dataSet: List<Any>)

    companion object {

        // This defines the view name that is an argument value in the go() in impl.
        val VIEW_NAME = "ReportBarChartComponentView"
    }

}

